composer.json

```
@@ -24,14 +24,13 @@
         "symfony/console": "^5.3||^6||^7"
     },
     "require-dev": {
-        "friendsofphp/php-cs-fixer": "3.51.0",
+        "friendsofphp/php-cs-fixer": "3.64.0",
         "guzzlehttp/guzzle": "^7.8.1",
         "pear/http_request2": "2.6.0",
         "phpmd/phpmd": "2.15.0",
-        "phpstan/phpstan": "1.10.59",
-        "phpunit/phpunit": "10.5.11",
-        "phing/phing": "3.0",
-        "squizlabs/php_codesniffer": "3.9.0"
+        "phpstan/phpstan": "1.12.7",
+        "phpunit/phpunit": "10.5.36",
+        "phing/phing": "3.0"
```

I think a line got lost by accident:

```suggestion
        "phing/phing": "3.0",
        "squizlabs/php_codesniffer": "3.9.0"
```